course/templates/course/_course_menu.html

```diff
@@ -41,7 +41,7 @@ <h4>{% trans "Course" %}</h4>
 </li>
 {% else %}
 <li role="presentation" class="header">
-	<h4>{% trans "Course" %}</h4>
+	<h4 id="course-menu-heading">{% trans "Course" %}</h4>
```

Comment:
Could you tell me why are you using the same id for all those three elements?. I did not understand these changes.

Reply:
If you look at the file as a whole, there is an `if` on line 16 which has the part including lines 20 and the part including line 44 as two options. It renders two different things depending on whether the course is multilingual or not. As a result, the the template only renders one of these options.

The on included on line 172 was added mistakenly, and has now been removed.

Reply:
mmm, this comment was published before I finished the whole review. Anycase, it looks fine now.